vsintegration/tests/FSharp.Editor.Tests/CodeFixes/UseTripleQuotedInterpolationTests.fs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+module FSharp.Editor.Tests.CodeFixes.UseTripleQuotedInterpolationTests
+
+open Microsoft.VisualStudio.FSharp.Editor
+open Xunit
+
+open CodeFixTestFramework
+
+let private codeFix = UseTripleQuotedInterpolationCodeFixProvider()
+let private diagnostic = 3373 // ... invalid interpolated string ...
+
```

Comment:
@abonie I need your help here:
1. Can you think of a more meaningful example?
2. Can I actually write the expected string in a simpler manner? 

We are changing 
`let answer = $"{"42"}"`
to this
`let answer = $"""{"42"}"""`

Reply:
`LazyLoadOption.className $"""{C.ExpandableThumbnail} {if isSmall then "" else C.TwiceFontsize}"""`
`| DueInDays x -> str $"""Review in {x} {naivePluralize x "day"}"""`

Reply:
Regarding question 2), we unfortunately don't have a nice way to deal with multiple (or more than 2x specifically) `"` chars in strings, we didn't implement that part of C#'s raw strings.
But in this case it would be more readable to just use single quoted string instead of triple quoted string:
```fsharp
FixedCode = "let answer = $\"\"\"{\"42\"}\"\"\""
```
You can add `\n` for newlines if you need them or you can actually have multi-line literals with single quotes too.

Reply:
Regarding 1), I am not sure what you mean by more meaningful. I think this is a good example of what the code fix does; from readability or tests-as-documentation perspective, I wouldn't complicate it further.

In terms of catching some edge cases etc., I am not sure. Would something like `$"{"42" + "0"}"` be worthwhile to test?

Reply:
Another possible edge case (I don't really know how code-fixes work, so might not be an edge case at all) would be something like `$"{$"{42+0}"}"`. Also, might be interesting to check what happens with `$"{$"{"42"}"}"`, but idk what should happen there, because this is not a straightforward fix (we don't allow 3+ levels of string nesting, you have to use some variable or other expression instead of the inner string literal)

Reply:
Yeah I just meant something a bit more real world :)

So I am using your example @kerams's (thanks) and your approach to writing strings @abonie's (thanks).
I think the current test case covers most of the above - and works.